### ISSUE TITLE
Add battery conservation mode support

### ISSUE THREAD

**Issue by andersmmg** [d4nj1/TLPUI#102]
I use these commands on my Ideapad laptop, and it'd be really nice to have support for them here. Just a toggle switch for conservation mode if it's supported on a device and the thresholds are not.
Enable conservation mode:
`sudo tlp setcharge 0 0` (only charge to 60%)
Disable conservation mode:
`sudo tlp setcharge 0 1` (charge fully)

These laptops have this basic support from the firmware, but setting custom thresholds is not possible on this hardware.

**Comment by d4nj1** [comment]
Hi,
I have to look deeper into the charge config but I guess the values should always be in between 0 and 100. I think your first example would probably only load until 1% and not 60%. Correct me if I'm wrong.
In general it should be possible to add such a functionality where you could execute setcharge from UI. Let me also think about this a bit deeper.

**Comment by andersmmg** [comment]
In this example, my hardware only supports toggling the limiter, not setting the percentage. I believe it's hardware locked to 60%, and changing that setting works like a boolean value, whether to limit charge or not. I'm not totally sure what info you'd need to detect this functionality, let me know if you need to see the output of any commands to check stuff!

**Comment by d4nj1** [comment]
Yes, the output for `tlp-stat -b` would be great.

My current device only has this:
```
/sys/class/power_supply/BAT0/charge_control_start_threshold = (not available) 
/sys/class/power_supply/BAT0/charge_control_end_threshold   = (not available) 
```
But I also have an old Thinkpad that should be supported. I probably could test it there.

**Comment by andersmmg** [comment]
Here's the output: http://sprunge.us/NjJD5h

**Comment by farinc** [comment]
I am not sure if its just on my end, but I have recently started using TLP 1.4 and TLPUI 1.5 and I cannot lower the value of `STOP_CHARGE_THRESH_BAT0 `to below 5 in the ui. I can otherwise influence this using just TLP functionality. This issue also appears to be related to the older issue #99. 

I agree with @andersmmg that a toggle or something would be a nice functionality.

Here is my tlp-stat -b output, [https://gist.github.com/farinc/1bb837ca40bf1266f6365351c8d2e873](url)

**Comment by d4nj1** [comment]
@farinc Yes, I understand that the default values are quite broken as this configuration is somewhat a mix between true/false or min/max value depending on the supported hardware. I am currently unsure how to handle this in a good manner.
Please be patient as I am not able to invest lots of my spare time at the moment.

**Comment by anaximeno** [comment]
Just a bit more context for this https://gist.github.com/linrunner/4a6876648765fac5e141f15d0582a945

**Comment by simonesestito** [comment]
> I cannot lower the value of STOP_CHARGE_THRESH_BAT0 to below 5 in the ui

I have an Ideapad laptop too and I can confirm it is still the case in TLP UI version 1.6.1

**Comment by anaximeno** [comment]
@d4nj1 Wouldn't this merge request https://github.com/d4nj1/TLPUI/pull/134 be enough to solve the problem?

**Comment by d4nj1** [comment]
Hi @anaximeno  and @simonesestito ,
please check if #146 is what you expect.

Thanks in advance

**Comment by d4nj1** [comment]
Fixed in #146 and released in https://github.com/d4nj1/TLPUI/releases/tag/tlpui-1.6.4
